cmake/projects/PocoCpp/hunter.cmake

```diff
@@ -4,6 +4,17 @@ include(hunter_pick_scheme)
 include(hunter_cacheable)
 include(hunter_download)
 
+hunter_add_version(
+    PACKAGE_NAME
+    PocoCpp
+    VERSION
+    1.8.1-p0
```

Comment:
Real tag mismatch, should be `1.8.1-p1`.

Reply:
Oh, we never released `1.8.1-p0`, so I thought the first release has `p0` and then so on.
Is there a naming rule or something?

Reply:
> Is there a naming rule or something?

I think it's clean and convenient approach when upstream version matches version in Hunter. What is the point of introducing different version?

Reply:
I understand your point but if we release `1.8.1-p1`, someone might search for `1.8.1-p0` (which we will be skipping). Hunter-packages poco versions are after all internal and abstracted from users using Hunter.

I am good with upstream version <-> Hunter version as well. Will do and squash.

Reply:
>I am good with upstream version <-> Hunter version as well. Will do and squash.

Done